### DISCUSSION TITLE
json file and cache

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#325] [Answers & Questions] [answered]
when i reupload a json with new content, cables sticks to the old version, even when i reload the patch/editor. only full cache reset/reload showed the recent iteration of the json file. anything i can do about this ? how would you approach working with a local json file which gets edited a lot. maybe edit it in cables somehow ?

**Comment by steam0r** [comment, marked as answer]
could you concatenade some random get parameter to the asset url and put it into ajaxrequest (something like ?t=SOMEKINGOFTIMESTAMP)?

**Reply by wirmachenbunt** [reply]
mmm, sounds like a dirty trick :) i switched to the string editor, editing the json in cables, that's much easier
